vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data.cc

```diff
@@ -192,6 +207,10 @@ void VectorData::Normalize() {
   }
 }
 
+float VectorData::ComputeSimilarity(const VectorData& v_sim) const {
+  return 0.5;
```

Comment:
Just a reminder to fill this in. You'll probably want to add a unit test for it as well in the vector data unittest file